src/Accelerators/NNPA/Runtime/CMakeLists.txt

```diff
@@ -21,6 +21,6 @@ set_target_properties(RuntimeNNPA
   PROPERTIES
   LANGUAGE C
   POSITION_INDEPENDENT_CODE TRUE
-  COMPILE_OPTIONS "-O3;-fopenmp"
+  COMPILE_OPTIONS -option O3 fopenmp
```

Comment:
@chentong319 I added the `-option` to bypass this error I was seeing when compiling `onnx-mlir`.

`clang: error: unsupported option '-fopenmp'`

I got the idea to use option from the following documentation: https://cmake.org/cmake/help/latest/command/target_compile_options.html

Reply:
Or I can remove `-fopenmp` .... it is blocking my building

Reply:
Yes, you can remove fopenmp flag now.
Which build failed with this flag? This flag will be needed in future.